src/overlays/actors/ovl_Bg_Jya_Bigmirror/z_bg_jya_bigmirror.c

```diff
@@ -26,21 +28,235 @@ const ActorInit Bg_Jya_Bigmirror_InitVars = {
     (ActorFunc)BgJyaBigmirror_Update,
     (ActorFunc)BgJyaBigmirror_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Jya_Bigmirror/func_808936E0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Jya_Bigmirror/func_80893750.s")
+typedef struct {
+    /* 0x00 */ Vec3f pos;
+    /* 0x0C */ s16 params;
+    /* 0x0E */ s16 solvedRotY;
+    /* 0x10 */ s16 initRotY;
+} BigMirrorDataEntry; // size = 0x14
+
+static BigMirrorDataEntry sCobraSpawnData[] = {
+    { { -560, 1743, -310 }, 0xFF01, 0x4000, 0x8000 },
+    { { 60, 1743, -310 }, 0xFF02, 0x8000, 0xA000 },
```

Comment:
Would be nice to have .0f in those float vectors.
Also most of these aren't technically s16s, so I don't know how we feel about that.

Reply:
Yeah, no way around calling them s16s since they are explicitly lh in the function that uses them. I'll do the floats, though, not sure how I missed those

Reply:
To be clear, I wasn't saying they weren't actually s16, I just meant the values aren't technically valid for a s16